lib/core-integration/src/Test/Integration/Scenario/API/Shelley/CoinSelections.hs

```diff
@@ -293,6 +309,74 @@ spec = describe "SHELLEY_COIN_SELECTION" $ do
                     (isValidDerivationPath purposeBIP44 . view #derivationPath))
             ]
 
+    it "WALLETS_COIN_SELECTION_07 - \
+        \Single output with excessively high token quantity." $
+        wallets_coin_selection_07 @n
+
+    it "WALLETS_COIN_SELECTION_08 - \
+        \Single output with excessively high number of assets." $
+        wallets_coin_selection_08 @n
+
+-- Attempt to create a coin selection with an output that has an excessively
+-- high token quantity. (This should fail.)
+--
+wallets_coin_selection_07
```

Comment:
Let's stick to the style of having the scenarios inline in the Spec?

Reply:
> Let's stick to the style of having the scenarios inline in the Spec?

I did it this way because including the scenarios inline involved a great deal of excessive wrapping (8 characters are already eaten up by the indentation).

But I don't feel strongly about it: I can re-inline them if you feel strongly about it. 👍🏻

Reply:
Fixed in 62944743aafa344b1b09935da53f900436818c13.